packages/gatsby/src/state-machines/develop/index.ts

```diff
@@ -106,6 +109,9 @@ const developConfig: MachineConfig<IBuildContext, any, AnyEventObject> = {
         ADD_NODE_MUTATION: {
           actions: [`markNodesDirty`, `callApi`],
         },
+        QUERY_RUN_REQUESTED: {
+          actions: [forwardTo(`run-queries`)],
```

Comment:
nit: there's no need for an array if there's just one action

Reply:
adjusted in https://github.com/gatsbyjs/gatsby/pull/28111/commits/0c24d1394d7dbc6bdaa879f423e2366e26881635